test/integration/scheduler/podsready/suite_test.go

```diff
@@ -76,7 +81,15 @@ var _ = ginkgo.AfterSuite(func() {
 })
 
 func managerAndSchedulerSetup(mgr manager.Manager, ctx context.Context) {
-	waitForPodsReady := true
+	cfg := config.Configuration{
+		WaitForPodsReady: &config.WaitForPodsReady{
+			Enable:  true,
+			Timeout: &metav1.Duration{Duration: 2 * time.Minute},
+		},
+	}
+
+	waitForPodsReady := waitForPodsReady(&cfg)
```

Comment:
isn't this just true?

Reply:
Yes, just wanted to have a single place to specify true/false, but cleaned up now.